src/components/Toast.scss

```diff
@@ -0,0 +1,32 @@
+@import "../css/_variables";
+
+.excalidraw {
+  .Toast {
+    box-sizing: border-box;
+    position: fixed;
+    bottom: 10px;
+    left: 50%;
+    margin-left: -150px;
+    z-index: 999999;
+    animation: fade-in 0.5s;
+    background-color: $oc-gray-5;
+    padding: 4px 0;
+    border-radius: 4px;
+    width: 300px;
+    text-align: center;
+  }
+
+  .Toast__message {
+    font-size: 1rem;
```

Comment:
is that needed? also rem.

Reply:
Just realized that it is redundant. Removed now!